src/energyplus/ForwardTranslator.cpp

```diff
@@ -3669,6 +3669,14 @@ namespace energyplus {
         translateAirflowNetworkHorizontalOpening(modelObject);
       }
 
+      // Specified Flow Rate
+      std::vector<model::AirflowNetworkSpecifiedFlowRate> sfrs = model.getConcreteModelObjects<model::AirflowNetworkSpecifiedFlowRate>();
+      std::sort(sfrs.begin(), sfrs.end(), WorkspaceObjectNameLess());
+      for (auto modelObject : sfrs) {
```

Comment:
(I know you're just doing the same as the above, but this makes a copy on each iteration. it's fine here, we'll fix in a soonish PR for performance)